src/foundry/client/apps/forms/grid-config.d.mts

```diff
@@ -134,5 +134,11 @@ declare global {
       "background.offsetX": Scene["background"]["offsetX"];
       "background.offsetY": Scene["background"]["offsetY"];
     }
+
+    interface GridConfigData {
+      gridTypes: Record<foundry.CONST.GRID_TYPES, string>;
+      scale: number;
+      scene: Scene;
```

Comment:
I'm actually not all that sure what the type should be here.

It looks like on the `GridConfig` class, we're specifying that the second type parameter of `GridConfig`'s parent `FormApplication` is `InstanceType<ConfiguredDocumentClass<typeof Scene>>`; which seems to evaluate to `Scene`? It also looks like in the `GridConfig` namespace, we're using `Scene` directly. So, I went with `Scene`. But if this should be something else, I can change it to that.

Is this right to be `Scene`? Or, should it be `InstanceType<ConfiguredDocumentClass<typeof Scene>>`, or something else?

Reply:
I'm not at all familiar with the API of Foundry, or modifying it; and I've only been using it for a few months. So I don't _really_ get how that works, but this is good context to have. Thanks for explaining it!

I'll make that change.